app/containers/HomePage/index.js

```diff
@@ -130,4 +131,4 @@ export default compose(
   withReducer,
   withSaga,
   withConnect,
-)(HomePage);
+)(toJS(HomePage));
```

Comment:
http://redux.js.org/docs/recipes/UsingImmutableJS.html

> Note that Immutable.JS objects do have a toJS() method, which returns the data as a plain JavaScript data structure, but this method is extremely slow, and using it extensively will negate the performance benefits that Immutable.JS provides

Reply:
This is calling the HOC. This pattern is taken directly from the docs you linked at the [bottom of the page](http://redux.js.org/docs/recipes/UsingImmutableJS.html#use-a-higher-order-component-to-convert-your-smart-components-immutablejs-props-to-your-dumb-components-javascript-props)